Code/Framework/AzNetworking/AzNetworking/Utilities/EncryptionCommon.cpp

```diff
@@ -537,7 +537,13 @@ namespace AzNetworking
         SSL_CTX_set_cookie_verify_cb(context, VerifyCookieCallback);
 
         // Automatically generate parameters for elliptic-curve diffie-hellman (i.e. curve type and coefficients).
+
+        // note that the below generates a warning depending on your verison of openssl - on some, these functions
```

Comment:
typo: verison -> version
```suggestion
        // note that the below generates a warning depending on your version of openssl - on some, these functions
```